### PR TITLE
sys-apps/bleachbit: Bump to 3.0

### PR DESCRIPTION
**Pull request by tietzc**
Package-Manager: Portage-2.3.76, Repoman-2.3.16
Signed-off-by: Christian Tietz <christian.tietz@mailbox.org>

### CODE REVIEW

**Comment by gentoo-bot2** [comment]
## Pull Request assignment

*Submitter*: @tietzc
*Areas affected*: ebuilds
*Packages affected*: sys-apps/bleachbit

**sys-apps/bleachbit**: @tietzc, @gentoo/proxy-maint

## Linked bugs

No bugs to link found. If your pull request references any of the Gentoo bug reports, please add appropriate [GLEP 66](https://www.gentoo.org/glep/glep-0066.html#commit-messages) tags to the commit message and request reassignment.

---
In order to force reassignment and/or bug reference scan, please append `[please reassign]` to the pull request title.

*Docs*: [Code of Conduct](https://wiki.gentoo.org/wiki/Project:Council/Code_of_conduct) ● [Copyright policy](https://www.gentoo.org/glep/glep-0076.html) ([expl.](https://dev.gentoo.org/~mgorny/articles/new-gentoo-copyright-policy-explained.html)) ● [Devmanual](https://devmanual.gentoo.org/) ● [GitHub PRs](https://wiki.gentoo.org/wiki/Project:GitHub/Pull_requests) ● [Proxy-maint guide](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide)

File: sys-apps/bleachbit/bleachbit-3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+gtk"
+
+RDEPEND="
+	dev-python/chardet[$PYTHON_USEDEP]
+	dev-python/scandir[$PYTHON_USEDEP]
+	gtk? ( dev-python/pygobject:3[$PYTHON_USEDEP] )
```
Comment (juippis) [inline sys-apps/bleachbit/bleachbit-3.0.ebuild]: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Policies#USE-Controlled_Optional_RDEPENDs
You should use `optfeature` or similar instead.

File: sys-apps/bleachbit/bleachbit-3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+"
+BDEPEND="
+	dev-python/setuptools[$PYTHON_USEDEP]
+	sys-devel/gettext
+"
+
+python_prepare_all() {
+	# choose correct Python implementation, bug #465254
+	sed -i 's/python/$(PYTHON)/g' po/Makefile || die
```
Comment (juippis) [inline sys-apps/bleachbit/bleachbit-3.0.ebuild]: I'm not saying this is incorrect, but maybe `${EPYTHON}` is what you're looking for here?

File: sys-apps/bleachbit/bleachbit-3.0.ebuild (hunk 1 above)
Comment (tietzc) [reply sys-apps/bleachbit/bleachbit-3.0.ebuild]: Virtually anyone would want a GUI for this. If it must not be controlled by a USE flag, my best bet would probably be to turn it into a hard dependency.

File: sys-apps/bleachbit/bleachbit-3.0.ebuild (hunk 2 above)
Comment (tietzc) [reply sys-apps/bleachbit/bleachbit-3.0.ebuild]: Indeed.

**Pull request closed without merging**